bibliography.bib

```diff
@@ -178,6 +178,35 @@ @InProceedings{clusterdata:Reiss2012
 
 
 
+################ 2023
+@INPROCEEDINGS{clusterdata:Tuns2023,
+  author = {Tuns, Adrian-Ioan and Spătaru, Adrian},
+  booktitle = {2023 25th International Symposium on Symbolic and Numeric Algorithms for Scientific Computing (SYNASC)},
+  title = {Cloud Service Failure Prediction on Google’s Borg Cluster Traces Using Traditional Machine Learning},
+  year = 2023,
+  month = Sep,
+  ISSN = {2470-881X},
+  doi = {10.1109/SYNASC61333.2023.00029},
+  url = {https://doi.org/10.1109/SYNASC61333.2023.00029},
+  pages = {162-169},
+  keywords = {Cloud computing;Machine learning algorithms;Scientific computing;Clustering algorithms;
+   Prediction algorithms;Boosting;Classification algorithms;failure prediction;big data;machine learning;
+   classification algorithms;Google Borg},
+  abstract = {The ability to predict failures in complex systems is crucial for maintaining their
+   optimal performance, opening the possibility of reducing downtime and minimizing costs.
+   In the context of cloud computing, cloud failure represents one of the most relevant problems,
+   which not only leads to substantial financial losses but also negatively impacts the productivity
+   of both industrial and end users. This paper presents a comprehensive study on the application of
+   failure prediction techniques, by exploring four machine learning algorithms, namely Decision Tree,
+   Random Forest, Gradient Boosting, and Logistic Regression.The research focuses on analyzing the
+   workload of an industrial set of clusters, provided as traces in Google’s Borg cluster workload traces.
+   The aim was to develop highly accurate predictive models for both job and task failures, a goal which
+   was achieved. A job classifier having a performance of 83.97% accuracy (Gradient Boosting) and a task
+   classifier of 98.79% accuracy performance (Decision Tree) were obtained.},
```

Comment:
Please check: I think you need to protect all "%"s with a backslash to stop latex treating as comments.